src/transbank.h

```diff
@@ -10,15 +10,17 @@
 #include "message.h"
 #include "responses.h"
 
-enum TbkReturn{
+enum TbkReturn
+{
     TBK_OK = 0,
     TBK_NOK = -1
 };
 
-extern enum TbkReturn open_port(char* portName, int baudrate);
-extern char* sale(int amount, int ticket, bool send_messages);
+extern enum TbkReturn open_port(char *portName, int baudrate);
+extern char *sale(int amount, int ticket, bool send_messages);
```

Comment:
This change is not related with the GetTotals, and should be separated, but first i think we should discuss if we need to change the pointer naming convention, since the rest of the code is using ìnt* i`

Reply:
This was formatting